src/app/components/harvest/widgets/validations.component.html

```diff
@@ -0,0 +1,99 @@
+<!-- TODO Show something when not meta review stage -->
+<ng-container *ngIf="isMetaReviewStage$ | async">
+  <h5>Instructions</h5>
+
+  <hr />
```

Comment:
Only use the `hr`s to separate different sections - don't separate headings from content